db/migrate/20150813115025_create_container_resource_quota.rb

```diff
@@ -0,0 +1,19 @@
+class CreateContainerResourceQuota < ActiveRecord::Migration
+  def change
+    create_table :container_quotas do |t|
+      t.string     :name
+      t.datetime   :creation_timestamp
+      t.string     :resource_version
+      t.string     :ems_ref
```

Comment:
In parse_base_item? I think :ems_ref originally contains object id as defined by the ems.
Since k8b generates those I can also use them as pkey.